PowerEditor/src/WinControls/shortcut/shortcut.cpp

```diff
@@ -502,7 +499,7 @@ void Accelerator::updateShortcuts()
 			_pAccelArray[offset].key = shortcuts[i].getKeyCombo()._key;
 
 			// Special extra handling for shortcuts shared by Incremental Find dialog
-			if (std::find(incrFindAccIds.begin(), incrFindAccIds.end(), shortcuts[i].getID()) != incrFindAccIds.end())
+			if (std::find(std::begin(incrFindAccIds), std::end(incrFindAccIds), shortcuts[i].getID()) != std::end(incrFindAccIds))

```

Comment:
Changed so it can work on plain C arrays. Equivalent functionality wise otherwise.